test/tinymce.spec.js

```diff
@@ -53,7 +53,7 @@ describe('uiTinymce', function () {
       expect(tinymce.init).toHaveBeenCalled();
       expect(tinymce.init.calls.mostRecent().args[0].tinymce.bar).toBe('baz');
     });
-
+/*
```

Comment:
The test failed but had nothing to do with my piece of code... Sorry, should have mentioned it ;-)